instrumentation/methods/README.md

```diff
@@ -0,0 +1,5 @@
+# Settings for the methods instrumentation
+
+| System property 	| Type 	| Default 	| Description 	|
+|-----------------	|------	|---------	|-------------	|
+| `otel.instrumentation.methods.include` | String| None | List of methods to include for tracing. An empty lists doesn't generate any TypeInstrumentation for muzzle to analyze. |
```

Comment:
The second sentence about muzzle is certainly confusing here. Maybe replace this explanation with a reference to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/manual-instrumentation.md#creating-spans-around-methods-with-otelinstrumentationmethodsinclude ?

Reply:
Fixed.